lib/secret_vault.ex

```diff
@@ -226,6 +226,47 @@ defmodule SecretVault do
     end
   end
 
+
+  @doc """
+  Helper macro for getting secrets in `config/runtime.exs` file.
+
+  ## Example
+
+  ```
+  # in `config/runtime.exs`
+  import Config
+  import SecretVault, only: [runtime_secret!: 2]
+
+  config :my_app, MyApp.Repo,
+    password: runtime_secret(:my_app, "database_password")
```

Comment:
There is a typo here. The `!` variant is being imported but the regular `runtime_secret/2` is being called.